pkg/entities/record.go

```diff
@@ -179,7 +179,14 @@ func (d *dataRecord) AddInfoElement(element *InfoElement, val interface{}) (uint
 		} else {
 			bytesToAppend[0] = 2
 		}
-	case DateTimeSeconds, DateTimeMilliseconds:
+	case DateTimeSeconds:
+		// We expect time to be given in int64 as unix time type in go
+		v, ok := val.(int64)
```

Comment:
Let's make the user give the seconds in uint32 rather than doing the conversion. Let's make ourselves independent go time package. What do you think?

Reply:
Makes sense to me. Thanks!